deployment/src/main/java/io/quarkiverse/fury/deployment/FuryProcessor.java

```diff
@@ -34,7 +34,7 @@ public void findSerializableClasses(
             CombinedIndexBuildItem combinedIndex, BuildProducer<FurySerializerBuildItem> pojoProducer) {
         combinedIndex.getIndex().getAnnotations(FURY_SERIALIZATION).stream()
                 .filter(annotation -> annotation.target().kind() == AnnotationTarget.Kind.CLASS)
-                .forEach(i -> pojoProducer.produce(new FurySerializerBuildItem(i.target().asClass())));
+                .forEach(i -> pojoProducer.produce(FurySerializerBuildItem.buildItem(i.target().asClass())));
```

Comment:
You mean `ThirdPartyFooConfig` can not be found by Jandex here?

Reply:
Yes, `ThirdPartyFooConfig` is annotated by `FurySerialization`, but `ThirdPartyFooConfig` is not used in tests

Reply:
Hmm, it wierd. I will take a look.

Reply:
Ha, there is typo in the `testSerializeThirdPartyBar`. I fixed it.

Reply:
My bad, thanks